lightkurve/targetpixelfile.py

```diff
@@ -98,42 +112,22 @@ def header(self):
     @property
     def ra(self):
         """Right Ascension of target ('RA_OBJ' header keyword)."""
-        try:
-            return self.header['RA_OBJ']
-        except KeyError:
-            return None
+        return self.get_keyword('RA_OBJ')
 
     @property
     def dec(self):
         """Declination of target ('DEC_OBJ' header keyword)."""
-        try:
-            return self.header['DEC_OBJ']
-        except KeyError:
-            return None
+        return self.get_keyword('DEC_OBJ')
 
     @property
     def column(self):
-        try:
-            out = self.hdu[1].header['1CRV5P']
-        except KeyError:
-            out = 0
-        # ensure output has a value
-        if isinstance(out, fits.card.Undefined):
-            return 0
-        else:
-            return out
+        """CCD pixel column number ('1CRV5P' header keyword)."""
+        return self.get_keyword('1CRV5P', hdu=1, default=0)
```

Comment:
This is a much more elegant way of handling missing header values. Nice!

Reply:
I should have made a separate PR for this, but yes, let's use the same logic for all keywords by having a single `get_keyword` method!